src/Native/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -364,6 +364,16 @@ enum class SSL_DataHashAlgorithm : int64_t
 #endif
 };
 
+class SSL_DataHashSize 
```

Comment:
The same reason you would use an enum in C# here instead of a static class with constant values.
http://stackoverflow.com/questions/4709175/what-are-enums-and-why-are-they-useful